### PR TITLE
BabyAI 1.1

### PR DESCRIPTION
**Pull request by dyth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,16 @@
...
-- [Replicating ICLR19 Results](#replicating-iclr19-results)
-- [Installation](#installation)
-- [Usage](#usage)
-- [Codebase Structure](docs/codebase.md)
-- [Levels](#the-levels)
-- [Training and Evaluation](docs/train-eval.md)
-- [Contributing](CONTRIBUTING.md)
-- [Troubleshooting](docs/troubleshooting.md)
-- [About](#about-this-project)
+To replicate or compare against our current baseline results, we recommend you use the [BabyAI 1.1 branch](TODO) and cite both:
```
[the remark this answers is not in the document]
Comment (dyth) [reply README.md:7]: The master branch is subject to change.  If we keep the BabyAI 1.1 branch, people can try to reproduce the results of the paper.  I will add an explanation about the changing master branch.

File: README.md [hunk 2]
```diff
@@ -30,31 +23,16 @@ If you use this platform in your research, please cite:
 }
 ```
 
-## Replicating ICLR19 Results
+This README covers instructions for [installation](##installation) and [troubleshooting](##troubleshooting).  Other instructions are:
 
-The master branch of this repository is updated frequently. If you are looking to replicate or compare against the results from the [ICLR19 BabyAI paper](https://openreview.net/forum?id=rJeXCo0cYX), please use the docker image, demonstration dataset and source code from the [iclr19 branch](https://github.com/mila-iqia/babyai/tree/iclr19) of this repository.
+- [Instructions on how to contribute](CONTRIBUTING.md)
```
[the remark this answers is not in the document]
Comment (dyth) [reply README.md:28]: Agreed, as mentioned above.

File: scripts/il_perf.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+from babyai import plotting as bp
+
+
+parser = argparse.ArgumentParser("Analyze performance of imitation learning")
+parser.add_argument("--path", default='.',
+    help="path to model logs")
+parser.add_argument("--regex", default='.*',
+    help="filter out some logs")
+parser.add_argument("--ttest", default=None,
```
[the remark this answers is not in the document]
Comment (dyth) [reply scripts/il_perf.py:18]: Done, also changed similar arguments in `rl_dataeff.py`

File: babyai/batchsampler.py [hunk 4]
```diff
@@ -1,58 +0,0 @@
...
-import copy
-
-class BatchSampler(object):
-    """
-    Class used to sample a batch of demonstrations from demonstrations of multiple
-    environments based on a distribution.
-    Used for Teacher Student Curriculum setting in imitation learning.
-    """
-
-    def __init__(self, demos, batch_size, seed, no_mem=False):
```
[the remark this answers is not in the document]
Comment (dyth) [reply babyai/batchsampler.py:11]: This class is never used on either master or ICLR-19 branches

**Comment by maximecb** [comment]
Looks good David 👍

**Comment by dyth** [comment]
Thanks @maximecb  We didn't have PyQT in `setup.py` or `environment.yaml`, so I'm assuming that's a minigrid thing.  In any case, I've removed the PyQT sections in `README.md`

**Comment by dyth** [comment]
@rizar My tests to reproduce the BabyAI 1.1 benchmarks have been successful.  They consist of:

1. RL, small arch, GoToRedBallGrey: `original`, `original_endpool`, `original_endpool_res`, `bow_endpool_res`, `pixels_endpool_res`
2. IL, small arch, GoToRedBallGrey: `bow`, `bow_endpool`, `bow_endpool_res`

**Comment by dyth** [comment]
RL sample efficiency:
../beluga/logs/rl/GoToRedBallGrey/original_test 159999.0
../beluga/logs/rl/GoToRedBallGrey/original_endpool_test 164785.0
../beluga/logs/rl/GoToRedBallGrey/pixels_endpool_res_test 163982.0
../beluga/logs/rl/GoToRedBallGrey/bow_endpool_res_test 164538.0
../beluga/logs/rl/GoToRedBallGrey/original_endpool_res_test 162186.0

IL performance:
../beluga/logs/il/GoTo/original_endpool_res/10000_test 0.80322265625
../beluga/logs/il/GoTo/bow_endpool_res/10000_test 0.95671875
../beluga/logs/il/GoTo/original/10000_test 0.67263671875
../beluga/logs/il/GoToRedBallGrey/original_endpool_res/5000_test 0.9903515625
../beluga/logs/il/GoToRedBallGrey/bow_endpool_res/5000_test 0.9926953125
../beluga/logs/il/GoToRedBallGrey/original/5000_test 0.99029296875

**Pull request merged**